### PR TITLE
[KSU Migration] Changes for icon, visual and description

### PR DESCRIPTION
**Pull request by justin-a30**
Changed
- name ✓
- description ✓
- icon ✓
- string in updaterscript ✓

### CODE REVIEW

**Comment by Aryanxfa** [comment]
Hi justin, thanks for the pr. can you remove the magisk option and its corresponding dependencies as well
Thanks!

**Comment by justin-a30** [comment]
Hmm? Remove it? I guess no, as still many users love magisk.

**Comment by Aryanxfa** [comment]
> Hmm? Remove it? I guess no, as still many users love magisk.

I have been planning on it for a while. We are stuck with older magisk and people can still install with zip

File: META-INF/com/google/android/updater-script [hunk 1]
```diff
@@ -348,23 +348,12 @@ ifelse(is_mounted("/product"), unmount("/product"));
...
-  package_extract_file("device/"+file_getprop("/tmp/aroma/compatible.prop", "device_id_alt")+"/dtbo.img", "/dev/block/platform/13500000.dwmmc0/by-name/dtbo");
-  #Magisk
-  package_extract_file("mods/magisk.zip", "/tmp/aroma/magisk.zip");
-  run_program("/tmp/busybox", "unzip", "/tmp/aroma/magisk.zip", "META-INF/com/google/android/*", "-d", "/tmp/aroma/");
-  run_program("/tmp/busybox", "sh", "/tmp/aroma/META-INF/com/google/android/update-binary", "dummy", "1", "/tmp/aroma/magisk.zip");
-  delete_recursive("/tmp/magisk.zip");
-endif;
-if file_getprop("/tmp/aroma/kernel.prop","selected") == "2" then
-  ui_print("Installing Vanilla Kernel...");
+  ui_print("Installing Eureka KernelSU...");
```
Comment (Aryanxfa) [inline META-INF/com/google/android/updater-script]: add @> to bold this line

File: META-INF/com/google/android/aroma/install_custom.edify [hunk 2]
```diff
@@ -57,12 +57,11 @@ endif;
 #KERNEL
 ###############################################
 menubox(
-    "Root Solution",
+    "Kernel Option",
     "Select your Kernel type, installs Eureka kernel and DTB 5",
-    "@su", 
+    "@kern",
```
Comment (Aryanxfa) [inline META-INF/com/google/android/aroma/install_custom.edify]: there is no icon named kern. why changing it ?

Comment (justin-a30) [reply META-INF/com/google/android/aroma/install_custom.edify]: what are you talking about? i didnt change it.

**Review by Aryanxfa — APPROVED** [review]

**Pull request merged**